source/content/guides/wordpress-composer/08-faq-integrated-composer-wp.md

```diff
@@ -0,0 +1,86 @@
+---
+title: WordPress with Integrated Composer on Pantheon
+subtitle: Integrated Composer with WordPress FAQ
+description: Get answers to your Integrated Composer with WordPress questions.
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/faq-integrated-composer-wp
+anchorid: faq-integrated-composer-wp
+---
+
+This section provides questions to frequently asked Integrated Composer with WordPress and Bedrock questions.
+
+## Is Integrated Composer for WordPress using Bedrock compatible with Multisite?
+
+Bedrock is compatible with [Multisite](/guides/multisite/). However, you might need to do extra work to ensure that admin URLs work correctly on subdomain installs. You can use the `roots/multisite-url-fixer`  mu-plugin to assist you with this work. Run the command below in your Bedrock directory:
+
+```bash{promptUser: user}
+composer require roots/multisite-url-fixer
```

Comment:
I'd modify this section based on the [Known Issues](https://pantheon.io/docs/guides/wordpress-composer/wordpress-composer-managed#known-issues) with the WordPress Composer Managed upstream.